erpnext/stock/doctype/inventory_dimension/test_inventory_dimension.py

```diff
@@ -447,7 +448,7 @@ def test_inter_transfer_return_against_inventory_dimension(self):
 				self.assertEqual(d.store, "Inter Transfer Store 2")
 
 	def test_validate_negative_stock_for_inventory_dimension(self):
-		frappe.local.inventory_dimensions = {}
+		get_inventory_dimensions.clear_cache()
```

Comment:
you want something like this ig, since whitelisting adds a function layer of its own

```suggestion
		get_inventory_dimensions.__wrapped__.clear_cache()
```

Reply:
or you can store that request cached function in a separate variable before whitelisting - and use that variable in tests

Reply:
Worth fixing in framework too IMO

Reply:
request_cache doesn't implement clear_cache lol